src/modules/fundingManager/bondingCurve/FM_BC_Bancor_Redeeming_VirtualSupply_v1.sol

```diff
@@ -598,4 +598,27 @@ contract FM_BC_Bancor_Redeeming_VirtualSupply_v1 is
             );
         }
     }
+
+    // -------------------------------------------------------------------------
+    // Overridden Internal Functions
+
+    /// @notice Handles issuance tokens by minting them to the receiver.
+    /// @param  _receiver The address that will receive the bought tokens.
+    /// @param  _issuanceMintAmount The amount of issuance tokens to handle.
+    function _handleIssuanceTokenAfterBuy(
```

Comment:
Still singular naming, while the other one is now plural. Change to plural as well.